src/main/java/org/cryptomator/hubcli/ListVaults.java

```diff
@@ -2,26 +2,56 @@
 
 import picocli.CommandLine;
 
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.concurrent.Callable;
+
 @CommandLine.Command(name = "list-vaults",
-        description = "List all vaults.")
-class ListVaults implements Runnable {
+        description = "List vaults.")
+class ListVaults implements Callable<Integer> {
+
+    @CommandLine.Mixin
+    Common common;
 
-    //TODO: ideas
-    // --format: [json | csv]
-    // --properties: list of properties to output
+    @CommandLine.Mixin
+    AccessToken accessToken;
 
-    @CommandLine.Option(names = { "--full" }, description = "Print all properties of each vault")
-    boolean fullInfo;
+    @CommandLine.Option(names = {"--all"}, description = "Print all vaults. Requires admin role")
+    boolean all;
```

Comment:
Since the cli user normally does not have the admin role, shouldn't we remove this option for the start?

Reply:
There may be cases where you want to get all the vaults. Since adding the `admin` role to the client is really easy, the implementation is already done, and does not add that much complexity, I would vote to have the option.

You would rather remove it?

Reply:
Removed in [e174a11](https://github.com/cryptomator/hub-cli/pull/2/commits/e174a11bc3b8abeeb711f86a6563d457c2b2011c)